processing/src/main/java/org/apache/druid/frame/read/columnar/StringFrameColumnReader.java

```diff
@@ -92,18 +85,20 @@ public Column readRACColumn(Frame frame)
     final Memory memory = frame.region(columnNumber);
     validate(memory);
 
+    if (isMultiValue(memory)) {
+      throw DruidException.defensive("Only Window Functions invoke readRACColumn and they do not support MVDs. "
```

Comment:
After this [PR](https://github.com/apache/druid/pull/17002) rejects the MVDs while processing, I guess this would not be invoked. But the msg is a suggesting invalid input. I think we can still make the exception as Invalid Input. will do it